smarty/templates/instrumentstatus_controlpanel.tpl

```diff
@@ -1,67 +1,84 @@
 <script type="text/javascript" src="{$baseurl}/js/instrument_controlpanel_control.js"></script>
-
+<script type="text/javascript" src="{$baseurl}/htdocs/vendor/sweetalert/sweetalert.js"></script>
+{literal}
+    <script type="text/javascript">
+        <!--
+        function swalFunction(event) {
```

Comment:
This should be in a javascript file of the module, not inline.

Reply:
Created a js file